src/index.js

```diff
@@ -1,38 +1,56 @@
-const todo = [
-  {
-    index: 1,
-    description: 'Take breakfast',
-    completed: true,
-  },
-  {
-    index: 2,
-    description: 'Attend morning session',
-    completed: true,
-  },
-  {
-    index: 3,
-    description: 'Work on the project',
-    completed: false,
-  },
-  {
-    index: 4,
-    description: 'Take lunch',
-    completed: false,
-  },
-];
-
-const list = document.querySelector('.list');
-let str = '';
-todo.map((activity) => {
-  const li = document.createElement('li');
-  li.setAttribute('class', 'list-item');
-  str = `<div>
-            <input type="checkbox" id=${activity.index} >
-            <label for="${activity.index}">${activity.description}</label>
-          </div>
-          <i class="fa-solid fa-ellipsis-vertical"></i>
-          `;
-  li.innerHTML = str;
-  list.appendChild(li);
-  return list;
+import { Todo } from './modules/storage.js';
+import './style.css';
+import './modules/fontawesome/css/all.css';
+
+const form = document.querySelector('.form');
+const inputField = document.querySelector('#activity');
+
+Todo.display();
+Todo.checkedTask();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const description = inputField.value;
+  const todo = new Todo(description);
+  todo.addTodo();
+});
```

Comment:
Submit action is not adding new Todo to your list as you can see in the video below 

https://user-images.githubusercontent.com/12239821/173189744-189e7bcc-2040-4ac4-808c-9c1fb36495f7.mp4


Kindly check that you are getting the right form and passing the action to it.

Reply:
Thanks, Trying using the enter button to submit the form data.  It would be better if you review my project at once

Reply:
Hello @Bornittah. I just tried using `Enter` to submit the form and it did submit. Kindly implement the submit using the button and submit another pull-request.